GPU_COMPUTE_CAPABILITY not defined

When attempting to compile on latest `main` without GPU, I get the following error:

```
../include/Macro.h:35:32: error: operator '>=' has no left operand
   35 | #if   ( GPU_COMPUTE_CAPABILITY >= 200  &&  GPU_COMPUTE_CAPABILITY < 300 )
      |                                ^~
../include/Macro.h:37:32: error: operator '>=' has no left operand
   37 | #elif ( GPU_COMPUTE_CAPABILITY >= 300  &&  GPU_COMPUTE_CAPABILITY < 500 )
      |                                ^~
../include/Macro.h:39:32: error: operator '>=' has no left operand
   39 | #elif ( GPU_COMPUTE_CAPABILITY >= 500  &&  GPU_COMPUTE_CAPABILITY < 600 )
      |                                ^~
../include/Macro.h:41:32: error: operator '>=' has no left operand
   41 | #elif ( GPU_COMPUTE_CAPABILITY >= 600  &&  GPU_COMPUTE_CAPABILITY < 700 )
      |                                ^~
../include/Macro.h:43:32: error: operator '>=' has no left operand
   43 | #elif ( GPU_COMPUTE_CAPABILITY >= 700  &&  GPU_COMPUTE_CAPABILITY < 750 )
      |                                ^~
../include/Macro.h:45:32: error: operator '>=' has no left operand
   45 | #elif ( GPU_COMPUTE_CAPABILITY >= 750  &&  GPU_COMPUTE_CAPABILITY < 800 )
      |                                ^~
../include/Macro.h:47:32: error: operator '>=' has no left operand
   47 | #elif ( GPU_COMPUTE_CAPABILITY >= 800  &&  GPU_COMPUTE_CAPABILITY < 890 )
      |                                ^~
```

--- a/include/Macro.h
+++ b/include/Macro.h
@@ -31,6 +31,7 @@
 #define TURING       6
 #define AMPERE       7
 
+#ifdef GPU
 #if   ( GPU_COMPUTE_CAPABILITY >= 200  &&  GPU_COMPUTE_CAPABILITY < 300 )
 # define GPU_ARCH FERMI
 #elif ( GPU_COMPUTE_CAPABILITY >= 300  &&  GPU_COMPUTE_CAPABILITY < 500 )
@@ -45,9 +46,10 @@
 # define GPU_ARCH TURING
 #elif ( GPU_COMPUTE_CAPABILITY >= 800  &&  GPU_COMPUTE_CAPABILITY < 890 )
 # define GPU_ARCH AMPERE
-#elif ( defined GPU )
+#else
 # error : ERROR : Unknown GPU_COMPUTE_CAPABILITY !!
-#endif
+#endif // GPU_COMPUTE_CAPABILITY
+#endif // #ifdef GPU
 
 
 // models

--- a/src/Auxiliary/Aux_TakeNote.cpp
+++ b/src/Auxiliary/Aux_TakeNote.cpp
@@ -366,7 +366,7 @@ void Aux_TakeNote()
       fprintf( Note, "GPU_ARCH                        UNKNOWN\n" );
 #     endif
       fprintf( Note, "GPU_COMPUTE_CAPABILITY          %d\n", GPU_COMPUTE_CAPABILITY );
-#     endif
+#     endif // #ifdef GPU
 
 #     ifdef LAOHU
       fprintf( Note, "LAOHU                           ON\n" );

